src/main/java/de/tum/in/www1/artemis/web/rest/dto/competency/CompetencyJolDTO.java

```diff
@@ -0,0 +1,18 @@
+package de.tum.in.www1.artemis.web.rest.dto.competency;
+
+import java.time.ZonedDateTime;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.competency.CompetencyJol;
+
+/**
+ * A DTO for the CompetencyJol entity.
+ */
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public record CompetencyJolDTO(long competencyId, short jolValue, ZonedDateTime judgementTime) {
```

Comment:
I either need the `mastery` score or a boolean that indicates that the current mastery has changed since the time of last judgement.